index.js

```diff
@@ -49,7 +49,7 @@ const babel = require('rollup-plugin-babel');
  *    global, but we can rely on the fact that Ember users are using modules.)
  */
 module.exports = {
-  name: 'ember-fetch',
+  name: require('./package').name,
```

Comment:
Can we just leave it 'ember-fetch', it's not necessary to require a whole package for its name.

Reply:
I prefer the new system :D

Also, the `package.json` is already required and this will just hit the cache...